### DISCUSSION TITLE
Getting the error Could not resolve entry module "index.html"

### DISCUSSION THREAD

**Discussion by EbenezerOyenuga** [firefly-iii/firefly-iii#9094] [Firefly III]
When trying to run npm run build I was getting the error npm ERR! Missing script: "build"

then I added the lines below in the package.json file:
"dev": "vite",
"build": "vite build"

Now I am getting the error: 
error during build:
Could not resolve entry module "index.html".
    at getRollupError (file:///var/www/html/firefly-iii/node_modules/rollup/dist/es/shared/parseAst.js:392:41)
    at error (file:///var/www/html/firefly-iii/node_modules/rollup/dist/es/shared/parseAst.js:388:42)
    at ModuleLoader.loadEntryModule (file:///var/www/html/firefly-iii/node_modules/rollup/dist/es/shared/node-entry.js:19092:20)
    at async Promise.all (index 0)

Kindly help

**Comment by JC5** [comment]
What would you like to see happen, exactly?

**Reply by EbenezerOyenuga** [reply]
Actually the challenge is I cannot run the project as it is not seeing the manifest.json file. So I was hoping that after running "npm install", I can run "npm run build". But when I run "npm run build" it gave me the error above.

**Reply by JC5** [reply]
The files are included in the release artifacts, if you need them: https://github.com/firefly-iii/firefly-iii/releases/tag/v6.1.19

However, you can generate them with:

```
npm run build --workspace=v1
npm run prod --workspace=v2
```

Depending on whether you want the v1 or the v2 layout.
